### DISCUSSION TITLE
Aarch 64

### DISCUSSION THREAD

**Discussion by wills721** [bluenviron/mediamtx#852] [Questions]
I use Raspberry 64bit.  I want to build rtsp-simple-server w/o Docker.  Can't seem to find a way to do that?

**Comment by aler9** [comment]
You can compile the server from source, without Docker, on the Raspberry Pi 64-bit by following instructions here:

https://github.com/aler9/rtsp-simple-server#compile-and-run-from-source

**Reply by photon-schiesser** [reply]
Hi. I have compiled from source using those instructions, but I am getting the following error while the server is running:
`2022/09/27 14:49:36 INF [path cam] [rpicamera source] ERR: server was compiled without support for the Raspberry Pi Camera`
I modified the `paths` in my YAML file as instructed.
I am running a 3B+ with Bullseye 64-bit.
Is there something else I need to do to build with RPi support?

**Reply by aler9** [reply]
Compilation instructions with Raspberry Pi camera support have been added to the README:

https://github.com/aler9/rtsp-simple-server#compile-and-run-from-source

[a comment by an automated account is not shown]
